common-rendering/src/components/accordion.tsx

```diff
@@ -0,0 +1,121 @@
+// ----- Imports ----- //
+
+import { css } from "@emotion/react";
+import type { SerializedStyles } from "@emotion/react";
+import { neutral, line, background } from "@guardian/src-foundations/palette";
+import { headline } from "@guardian/src-foundations/typography";
+import { remSpace } from "@guardian/src-foundations";
+import { focusHalo } from "@guardian/src-foundations/accessibility";
```

Comment:
Had no idea this was a thing! Amazing, thanks for teaching me about this 😊

Reply:
Ah - thank Marjan! I just moved it from her code in key events